ThunderCloud/QuizAreaSelectionViewController.swift

```diff
@@ -18,7 +18,13 @@ open class QuizAreaSelectionViewController: UIViewController, QuizQuestionViewCo
     
     fileprivate var circleLayer: CAShapeLayer?
     
-    open var circleColor: UIColor? = .white
+    fileprivate var circleShadowLayer: CAShapeLayer?
```

Comment:
Depending a bit on your answer to the remove-add question below. I think this is great opportunity to use a `lazy var` and configure it there. We can set the `path` later. 
I do this a little bit in `SeparatorTableViewCell` in Blood. 
If we are on a clock then tbh it's something we can look at another time.